spec/Cabin/SingleConfigurableLight.vspec

```diff
@@ -0,0 +1,29 @@
+#
+# (C) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+
+#
+# Generic specification for a light whose color and brightness can be configured.
+#
+
+IsLightOn:
+  type: actuator
+  datatype: boolean
+  description: Indicates whether the light is turned on. True = On, False = Off.
+
+Intensity:
+  type: actuator
+  datatype: uint8
+  unit: percent
+  min: 1
```

Comment:
How come we have 1 as minimum? Can they not be completely turned off?

Reply:
I had initially the same question. However, after some discussions, I understood the reason behind that decision.
The idea is that a light `Intensity` controller can be used to attenuate (i.e., dimmerize) the light. Therefore, its main purpose is not to serve as a switch, but rather to set the intensity. 

In this sense, for example, you might have a light configured with an intensity of 50%. This configuration can be stored for a particular user profile. Apart from that, you can have the switch `IsLightOn` to just turn it on/off without the need to remember the previous state of the intensity. 

That is why the `Intensity` has a minimum value of 1%, so that it is not used as a switch. Which means that you can set the intensity to 100%, but the light is off unless you turn on the switch `IsLightOn`.

Reply:
I understand. We could possibly add something in a comment, possibly something like "Minimum value not zero as on/off is controlled by the signal IsLightOn", there will likely be more that will have the same question.

Reply:
Assume there are a separate switch and a separate dial for intensity. The switch can be on, with the intensity zero. The switch can be off, with the intensity 50%.
 
No point in trying to represent or avoid representing the switch state in the intensity: they are independent. That is, don’t limit the intensity to anything else than 0-100%...